### PR TITLE
Feature/update to compose 1.5.0 and Kotlin 1.9.0

### PR DESCRIPTION
**Pull request by Saamyyy**
## what 
- update to compose `1.5.0` along with Kotlin `1.9.0`
- disable Dokka as it's not yet supporting 1.9.0 for kotlin

### CODE REVIEW

File: uisdk/src/test/java/tech/dojo/pay/uisdk/presentation/ui/result/mapper/PaymentResultViewEntityMapperTest.kt [hunk 1]
```diff
@@ -22,7 +22,7 @@ class PaymentResultViewEntityMapperTest {
             .willReturn("Successful Payment")
         val expected = PaymentResultState.SuccessfulResult(
             appBarTitle = "Successful Payment",
-            imageId = 2131230882,
+            imageId = 2131230888,
```
Comment (ruancorreadojo) [inline uisdk/src/test/java/tech/dojo/pay/uisdk/presentation/ui/result/mapper/PaymentResultViewEntityMapperTest.kt:25]: thing to update here in the future, we should never use static ids to check values in test

**Review by ruancorreadojo — APPROVED** [review]

File: build.gradle.kts [hunk 2]
```diff
@@ -5,10 +5,10 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.0")
-        classpath("com.android.tools.build:gradle:7.2.0")
-        classpath("org.jetbrains.dokka:dokka-gradle-plugin:1.5.0")
-        classpath("org.jetbrains.dokka:dokka-core:1.5.0")
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.9.0")
+        classpath("com.android.tools.build:gradle:7.4.1")
+//        classpath("org.jetbrains.dokka:dokka-gradle-plugin:1.8.20")
```
Comment (paul-dojo) [inline build.gradle.kts:10]: Can these be removed?

**Review by paul-dojo — APPROVED** [review]

File: uisdk/src/test/java/tech/dojo/pay/uisdk/presentation/ui/result/mapper/PaymentResultViewEntityMapperTest.kt (hunk 1 above)
Comment (Saamyyy) [reply uisdk/src/test/java/tech/dojo/pay/uisdk/presentation/ui/result/mapper/PaymentResultViewEntityMapperTest.kt:25]: will include this as part of data flow refactor  PR

File: build.gradle.kts (hunk 2 above)
Comment (Saamyyy) [reply build.gradle.kts:10]: this is for Dokka we will use it back once we got any response from them about Kotlin `1.9.0` support

**Pull request merged**